content/en/blog/2024/otel-cicd-sig/index.md

```diff
@@ -0,0 +1,145 @@
+
+### Artifacts for supply chain security, aligned with the SLSA specification
+
+The [artifact attribute namespace](https://opentelemetry.io/docs/specs/semconv/attributes-registry/artifact/) had multiple attributes for its first implementation. One key set of attributes within this namespace cover [attestations](https://slsa.dev/attestation-model) that closely align with the [SLSA](https://slsa.dev/spec/v1.0/about) model. This is really the first time a direct connection is being made between Observability and Software Supply Chain Security. Consider the following [supply chain threat model](https://slsa.dev/spec/v1.0/threats) defined by SLSA:
+![SLSA supply chain threat model diagram](SLSA-supply-chain-model.png)
+
+These new attributes for artifacts and attestations help observe the sequence of events modeled in the above diagram in real time. Really, the conventions that exist today and those that will be added in the future enable interoperability between core software delivery capabilities like security and platform engineering via observability semantics.
+
+## What’s Next for CI/CD Observability Working Group
+
+The first major milestone we shared above, was the merge of the OTEP for extending the semantic conventions with the new attributes, which is now part of the OpenTelemetry Semantic Conventions latest release.
+
+The other important milestone was [OTEP #258](https://github.com/open-telemetry/oteps/pull/258) for Environment Variable Context Propagation that was just approved and merged. This OTEP sets the ground for writing the specification.
+
+Since we’ve made progress on our initial milestones, we’ve updated the [CI/CD Observability SIG milestones for the remainder of 2024](https://github.com/open-telemetry/community/blob/main/projects/ci-cd.md). Our goal is to finish out as many of the defined milestones as possible by the end of the year. Notably, we’re focused on: 
+
+* Adding [metric conventions for version control systems](https://github.com/open-telemetry/semantic-conventions/pull/1383).
+* Building tracing prototypes in CICD systems. (eg. ArgoCD, GitHub, GitLab, Jenkins)
+* Getting [OTEP #258](https://github.com/open-telemetry/oteps/pull/258) ready for implementation for the addition to the specification.
+* Adding additional attributes to the registry covering more domains like:
+  * [Software outage incidents](https://github.com/open-telemetry/semantic-conventions/issues/1185)
+  * [System attributes around CI/CD runners](https://github.com/open-telemetry/semantic-conventions/issues/1184)
+* Beginning work on Trace and Event (log) signal specifics. (Building the bridge for interoperability between other specifications)
```

Comment:
```suggestion
* Beginning work on trace and event (log) signal specifics to build the bridge for interoperability between other specifications.
```

Reply:
Although trace and event are defined terms in OpenTelemetry, they are common nouns and should not be capitalized. To illustrate: the OpenTelemetry Collector is capitalized because it is a proper noun referring to a specific collector. If you refer to other collectors, the term is not capitalized.